geode-apis-compatible-with-redis/src/distributedTest/java/org/apache/geode/redis/internal/data/PartitionedRegionStatsUpdateTest.java

```diff
@@ -0,0 +1,396 @@
+
+    for (int i = 0; i < 10; i++) {
+      jedis1.hset(HASH_KEY, FIELD, "finalvalue");
+    }
+
+    assertThat(jedis1.hgetAll(HASH_KEY).size()).isEqualTo(1);
+
+    long server2FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server2);
+    long server1FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
+
+    assertThat(server1FinalDataStoreBytesInUse)
+        .isEqualTo(server2FinalDataStoreBytesInUse)
+        .isEqualTo(initialDataStoreBytesInUse);
+  }
+
+  @Test
+  @Ignore("confirm that bucket size stats are being calculated correctly before enabling")
+  public void should_showMembersAgreeUponUsedSetMemory_afterDeltaPropagation() {
+    jedis1.sadd(SET_KEY, "other"); // two sadds are required to force
+    jedis1.sadd(SET_KEY, "value"); // deserialization on both servers
+    // otherwise primary/secondary can disagree on size, and which server is primary varies
+
+    long initialDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server2);
+
+    for (int i = 0; i < 10; i++) {
+      jedis1.sadd(SET_KEY, "value");
+    }
+
+    assertThat(jedis1.scard(SET_KEY)).isEqualTo(2);
+
+    long server1FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
+    long server2FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server2);
+
+    assertThat(server1FinalDataStoreBytesInUse)
+        .isEqualTo(server2FinalDataStoreBytesInUse)
+        .isEqualTo(initialDataStoreBytesInUse);
+  }
+
+  @Test
+  @Ignore("confirm that bucket size stats are being calculated correctly before enabling")
+  public void should_showMembersAgreeUponUsedStringMemory_afterDeltaPropagation() {
+    jedis1.set(STRING_KEY, "12345"); // two sets are required to force
+    jedis1.set(STRING_KEY, "value"); // deserialization on both servers
+    // otherwise primary/secondary can disagree on size, and which server is primary varies
+
+    long initialDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
+
+    for (int i = 0; i < 10; i++) {
+      jedis1.append(STRING_KEY, "a");
+    }
+
+    assertThat(jedis1.exists(STRING_KEY)).isTrue();
+    assertThat(jedis2.exists(STRING_KEY)).isTrue();
+
+    assertThat(jedis1.get(STRING_KEY)).isEqualTo("valueaaaaaaaaaa");
+    assertThat(jedis2.get(STRING_KEY)).isEqualTo("valueaaaaaaaaaa");
+
+    long server1FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server1);
+    long server2FinalDataStoreBytesInUse =
+        clusterStartUpRule.getDataStoreBytesInUseForDataRegion(server2);
+
+    assertThat(server1FinalDataStoreBytesInUse)
+        .isEqualTo(initialDataStoreBytesInUse);
+
+    assertThat(server2FinalDataStoreBytesInUse)
+        .isEqualTo(initialDataStoreBytesInUse);
```

Comment:
This will currently never be true.  We could change the for loop to just do just set the key to `value` repeatedly. then make sure the STRING_KEY is still equal to `value`. Then do the same assertion we did for sets and hashes:  
```
    assertThat(server1FinalDataStoreBytesInUse)
        .isEqualTo(server2FinalDataStoreBytesInUse)
        .isEqualTo(initialDataStoreBytesInUse);
```

Unfortunately, it still doesn't pass though.